src/app/modules/gb-export-module/gb-export.component.ts

```diff
@@ -23,12 +23,11 @@ export class GbExportComponent implements OnInit, OnDestroy {
 
   private timer: Timer;
 
-  // TODO: fix export workflow:
   // 1. user defines cohort in cohort selection
-  // 2. user goes to export tab, variables on the left are selectable
-  // 3. based on cohort and variables, compose combination constraint to get data table
-  // 4. user arranges data table
-  // 5. user selects export data type, defines export job
+  // 2. user goes to export tab, the variables on the left become selectable
+  // 3. user arranges data table dimensions, retrieve the new data table
+  // 4. retrieve export data types
+  // 5. user create export job based on arranged data table and selected export data types
```

Comment:
as I mentioned above, the flow will be slightly different for data view != `dataTable`

Reply:
you mean check if it is surveyTable or external export, if it is, do not use data table as input param for running export jobs?

Reply:
yes, sounds good!